sdk/iOS/OoyalaSkinSDK/OOSkinViewController.h

```diff
@@ -22,6 +22,24 @@
 @property (nonatomic, readonly) NSString *version;
 @property (nonatomic, readonly) OOClosedCaptionsStyle *closedCaptionsDeviceStyle;
 
+/**
+ Programatically change the fullscreen mode of the player.
+ Use this only after setting the OOSkinViewController.view.frame. Using it before will result in unexpected results.
+ 
+ Here is an example on how to use this property:
+ @code
+ // assume the property skinController of type OOSkinViewController exists and is already instantiated.
+ // Also assume you have an IBOutlet to a self.videoView pointing to an existing view.
```

Comment:
Should we add some logic to handle this case? Like if the value of view.frame is not set before calling fullscreen, we ignore the fullscreen call?